lib/builder.js

```diff
@@ -104,13 +111,15 @@ Builder.prototype.build = function() {
           })
       })
   })
-  return promise
+  return promise;
 }
 
 // Destructor-like method. Cleanup is synchronous at the moment, but in the
 // future we might change it to return a promise.
 Builder.prototype.cleanup = function() {
+  this._canceled = true
   this.builderTmpDirCleanup()
```

Comment:
Oh. Wait, I think you're already saying this in your original comment (your list is a to-do list, yes?)...